### ISSUE TITLE
Feature Request: User Completion Status

### ISSUE THREAD

**Issue by Bovive** [axewater/sharewarez#48]
I think it would be cool if a user could mark their completion status for a game so that they could filter a library down to games they have not played or finished yet.

Completion Status:

- Unplayed (Default for a new game)
- Playing
- On Hold
- Abandoned
- Played
- Completed

**Comment by axewater** [comment]
this is a9nother) great idea. or maybe even sync their account with 'backloggery' or similar sites ?
ill check which completion statuses they use, that will make syncing in the future doable.

**Comment by axewater** [comment]
Status 'endless' could be good to have.
And 'on hold' will be 'backlog' making it more clear you intend to play it.

**Comment by Bovive** [comment]
Ya those would work well too.

**Comment by Bovive** [comment]
> backloggery

Good call. I never heard of them before but if they have an API to sync with it makes sense to follow their format.
